build.sbt

```diff
@@ -75,7 +75,7 @@ lazy val runtime = Project(id = gyremockRuntimeName, base = file("runtime"))
     libraryDependencies := Seq(
       "ch.qos.logback" % "logback-classic" % "1.4.14" % Runtime,
       "com.lightbend.akka.grpc" %% "akka-grpc-runtime" % akkaGrpcVersion,
-      "com.typesafe.akka" %% "akka-actor" % "2.6.20",
+      "com.typesafe.akka" %% "akka-actor" % "2.7.0",
```

Comment:
I create a ActorSystem in `test`, and error happens since different versions between `akka-actor` (2.6.20) and `akka-stream`(2.6.19 introduced by akka-grpc-runtime) and `akka-discovery` (2.6.19 introduced by akka-grpc-runtime). But we could specify them to use `2.6.20`

Reply:
i meant same for akka-grpc version, everything needs to stay the same for the license

Reply:
Could you pls help point out where I could find the version in the license? I could not find it in local file: LICENSE

Reply:
https://github.com/touchdown/gyremock/blob/main/LICENSE - https://www.lightbend.com/blog/why-we-are-changing-the-license-for-akka

Reply:
Revert it. And downgrade the `akka-actor` version. Maybe it is better to keep them in the same version?

Reply:
patch versions dont matter too much

Reply:
Yeah, you are right. Prefer downgrading here so that we can avoid the below error

> I create a ActorSystem in `test`, and error happens since different versions between `akka-actor` (2.6.20) and `akka-stream`(2.6.19 introduced by akka-grpc-runtime) and `akka-discovery` (2.6.19 introduced by akka-grpc-runtime). But we could specify them to use `2.6.20`

Reply:
u either do that, or specify 2.6.20 for all those deps akka-stream, akka-discovery